starter/class/connect-four.js

```diff
@@ -21,24 +21,88 @@ class ConnectFour {
     Screen.setGridlines(true);
 
     // Replace this with real commands
-    Screen.addCommand('t', 'test command (remove)', ConnectFour.testCommand);
+    // Screen.addCommand('t', 'test command (remove)', ConnectFour.testCommand);
+    Screen.addCommand('up', 'moves cursor up', this.cursor.up);
+    Screen.addCommand('Down', 'moves cursor down', this.cursor.down);
+    Screen.addCommand('left', 'move cursor left', this.cursor.left);
+    Screen.addCommand('right', 'moves cursor right', this.cursor.right);
+    Screen.addCommand('return', "places a move in the current position", this.placeMove)
 
     this.cursor.setBackgroundColor();
     Screen.render();
   }
+  // Return 'X' if player X wins
+  // Return 'O' if player O wins
+  // Return 'T' if the game is a tie
+  // Return false if the game has not ended
 
-  // Remove this
-  static testCommand() {
-    console.log("TEST COMMAND");
+  placeMove = () => {
+    if (Screen.grid[this.cursor.row][this.cursor.col] === " ") {
+      Screen.setTextColor(this.cursor.row, this.cursor.col, 'white');
+      Screen.setGrid(this.row, this.col, this.playerTurn);
+      Screen.render()
+      if (Cursor.checkWin(Screen.grid)) {
+        Cursor.endGame(Cursor.checkWin(Screen.grid));
+      }
+    }
   }
 
   static checkWin(grid) {
```

Comment:
Is there a reason why you chose this to be `static`?

Reply:
I'm not sure tbh. What difference would it have made if It wasn't static?

Reply:
you can't invoke your own functions with `this.checkWin` if it's static, i believe.

Reply:
ah i take that back, you can't call static methods from within a constructor.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static#calling_static_members_from_a_class_constructor_and_other_methods

Reply:
The reason i say this is: you may have difficulty with calling `this.checkWin()` in `placeMove()`